distribution/lib/Standard/Base/0.0.0-dev/src/Main.enso

```diff
@@ -60,7 +60,7 @@ export project.Warning.Warning
 
 from project.Data.Boolean export Boolean, True, False
 from project.Function export all
-from project.Data.Numbers export Number, Integer, Decimal
+from project.Data.Numbers export Number, Integer, Decimal, Nan
```

Comment:
Not sure whether this should be exported by default? We don't export `Small_Integer`, neither `Big_Integer`.

Reply:
cc @radeusgd